app/main/controllers/template/rtmedia-functions.php

```diff
@@ -4092,16 +4092,16 @@ function rtm_is_buddypress_activate() {
  */
 function rtmedia_get_upload_url( $request_uri ) {
 
-	$url        = esc_url( trailingslashit( $request_uri ) );
-	$slug_split = explode( '/', $url );
-	$slug_split = array_values( array_filter( $slug_split ) );
+	$url          = esc_url( trailingslashit( $request_uri ) );
+	$slug_split   = explode( '/', $url );
+	$slug_split   = array_values( array_filter( $slug_split ) );
+	$rtmedia_slug = '/' . RTMEDIA_MEDIA_SLUG;
 
 	// check position of media slug for end of the URL.
-	if ( is_array( $slug_split ) && ! empty( $slug_split ) && RTMEDIA_MEDIA_SLUG === $slug_split[ count( $slug_split ) - 1 ] ) {
+	if ( is_array( $slug_split ) && ! empty( $slug_split ) && false !== strrpos( $url, $rtmedia_slug ) ) {
```

Comment:
@BhargavBhandari90 Use variable to store output of this `strrpos( $url, $rtmedia_slug )`, so use can use it on the next lines.